### PR TITLE
make method non-static

### PR DESCRIPTION
**Pull request by XiXiaPdx**

### CODE REVIEW

**Comment by XiXiaPdx** [comment]
Cool!  One more small change tied with this.  Could you take a look? `getProjectPath` doesn't need to be static anymore either.

File: src/main/java/com/newrelic/agent/instrumentation/verify/AfterEvaluationAction.java [hunk 1]
```diff
@@ -98,15 +98,15 @@ public void execute(@NonNull Project project) {
...
+    public boolean projectRequiresVerification(Project project) {
         return project.getGradle().getStartParameter().getTaskNames().stream()
                 .filter(taskName -> taskName.endsWith(VERIFIER_TASK_NAME))
                 .map(taskName -> getProjectPath(project, taskName))
                 .filter(Objects::nonNull)
                 .anyMatch(projectName -> project.getProjectDir().getPath().startsWith(projectName));
     }
 
-    private static String getProjectPath(Project project, String taskName) {
+    public String getProjectPath(Project project, String taskName) {
```
Comment (XiXiaPdx) [inline src/main/java/com/newrelic/agent/instrumentation/verify/AfterEvaluationAction.java:109]: Static is no longer needed

Comment (jeffalder) [reply src/main/java/com/newrelic/agent/instrumentation/verify/AfterEvaluationAction.java:109]: Can it still be `private`, though? It doesn't appear to be called directly anywhere but here.

Comment (XiXiaPdx) [reply src/main/java/com/newrelic/agent/instrumentation/verify/AfterEvaluationAction.java]: good point, fixed!

**Pull request merged**